repose-aggregator/components/filters/ip-classification/src/main/scala/edazdarevic/commons/net/CIDRUtils.java

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
It doesn't matter if there's a java source in the scala source folder, scala can compile it. I didn't feel like separating the things since it was the only one. It's copy pasta, because I have no reason to re-type it? Where is it copy-pasta'd from? It handles all the use cases for CIDR notations, so there's no reason to re-implement it.